docs/getting-started.md

```diff
@@ -1,27 +1,17 @@
-# Tekton Pipelines Tutorial
+# Getting Started with Tekton Pipelines
 
-This tutorial uses a simple `Hello World` example to show you how to:
-- Create a `Task`
-- Create a `Pipeline` containing your `Tasks`
-- Use a `TaskRun` to instantiate and execute a `Task` outside of a `Pipeline`
-- Use a `PipelineRun` to instantiate and run a `Pipeline` containing your `Tasks`
+This guide will get you started with tekton pipelines with a simple `Hello World` tutorial.
```

Comment:
The bullet list above gives a clear overview of what the reader will accomplish in the tutorial. I think you should restore it, because the new sentence isn't very informative. You also need to capitalize "Tekton Pipelines".

Reply:
fixed on new commit

Reply:
I will resolve this if you are ok with the change I made.